kernel/bpf/test_bpf/runtest.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
+# Include rhts environment
+. /usr/bin/rhts-environment.sh
+. /usr/share/beakerlib/beakerlib.sh
+
+# Logic:
+# 1. Download the SRPM and install the SRPM
+# 2. Find the source tarball and decompress
+# 3. Extract the lib/test_bpf.c
+# 4. obj-m+= test_bpf.o in Makefile
+# 5. make -C /lib/modules/`uname -r`/build M=`pwd` modules
+# 6. insmod test_bpf.ko
+# 7. journalctl -k --full | sed "s/$(hostname)//g"
+# Expected result
+# Apr 25 04:30:10  kernel: test_bpf: Summary: 349 PASSED, 0 FAILED, [0/341 JIT'ed]
+
+
+DistName=`rpm -E %{?dist} | sed 's/[.0-9]//g'`
+DistVer=`rpm -E %{?dist} | sed 's/[^0-9]//g'`
+DUPARCH=`arch`
+KVer=`uname -r | awk -F '-' '{print $1}'`
+KDIST=`uname -r | sed "s/.$(arch)//g" | awk -F '.' '{print "."$NF}'`
+KBUILD=`uname -r | awk -F '-' '{print $2}' | sed "s/.$(arch)//g" | sed "s/${KDIST}//g"`
+KBuild=${KBuild:-${KBUILD}}
+KBuildPrefix=`echo ${KBuild} | awk -F '.' '{print $1}'`
+
+kernel_nvr=`uname -r | sed "s/.$(arch)//g"`
+KVARIANT=`rpm -q --queryformat '%{sourcerpm}\n' -qf /boot/config-$(uname -r) | sed "s/.srpm//g;s/.src.rpm//g;s/-${kernel_nvr}//g"`
+
+rlJournalStart
+    rlPhaseStartSetup
+        for i in libgcc glibc-static gcc gcc-c++ kernel-devel elfutils-libelf-devel binutils-devel libcap-devel ; do
+            yum install -y $i
```

Comment:
Could you please detect whether dnf or yum is available and use that instead of hardcoding yum? You can use similar code as the other tests are, e.g. https://github.com/CKI-project/tests-beaker/blob/master/networking/common/install.sh#L8

Reply:
Sure. I'll do it during working hours and rebase the PR.